scss/_patterns_navigation.scss

```diff
@@ -3,6 +3,18 @@
 $lightness-threshold: 70;
 
 @mixin vf-p-navigation {
+  .content-curtain {
```

Comment:
We will think about the name, but for now lets rename it to `p-navigation-overlay`